plugin/klogrus/klogrus.go

```diff
@@ -7,17 +7,33 @@ import (
 	"github.com/twmb/franz-go/pkg/kgo"
 )
 
+var (
+	_ FieldLogger = (*logrus.Logger)(nil)
+	_ kgo.Logger  = (*Logger)(nil)
+)
+
+// FieldLogger interface.
```

Comment:
Can you change this doc line to expand just a bit on what a FieldLogger is or why I'd want to use it / what it does?